### PR TITLE
Md linting fixes

### PR DESCRIPTION
**Pull request by VijaySharma815**

### CODE REVIEW

**Review by solarw — APPROVED** [review]
lgtm

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ jobs:
           - copyright-check
           - check-manifest
           - check_api_docs
+          - docs-check
```
Comment (5A11) [inline .github/workflows/workflow.yml:31]:
```suggestion
          - markdown-check
```

File: Makefile [hunk 2]
```diff
@@ -276,6 +277,7 @@ check:
 	$(MAKE) liccheck
 	$(MAKE) copyright-check
 	$(MAKE) test
+	$(MAKE) docs-check
```
Comment (5A11) [inline Makefile:280]:
```suggestion
	$(MAKE) markdown-check
```

File: docs/staking.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+- **Redelegating**: This is the process where you transfer your staked tokens from one validator to another. This can be for many reasons, such as better returns, more trustworthiness, etc.
+- **Undelegating**: While your tokens are staked, you cannot spend them or send them to other users. To regain access to them, you must undelegate them. When you initiate this process, the funds will be removed from the validator they were delegated to, and must be left to cool down for a period of time (for example 21 days). After this period, the funds are automatically released into the user's wallet.
 
 ## Actions
 
 `LedgerClient` provides useful utilities for interacting with the staking component of the network.
 
 !!! note
-    For simplicity, the staking methods do not have an option for specifying the `denom` field. This is because in almost all networks, there is only one staking denomination. Therefore, the denomination used is the one specified in the [`NetworkConfig`](connect-to-network.md) supplied to the `LedgerClient` object.
```
Comment (5A11) [inline docs/staking.md:12]: Incorrect

File: docs/staking.md [hunk 4]
```diff
@@ -50,13 +50,13 @@ tx.wait_to_complete()
 ```
 
 !!! note
-    The cool down is tracked for each invocation of undelegate action. So for example if you trigger 3 undelegate actions on 3 consecutive days. The first batch of tokens will become available 3 days before the final batch.
```
Comment (5A11) [inline docs/staking.md:53]: Incorrect

File: HISTORY.md [hunk 5]
```diff
@@ -2,102 +2,116 @@
...
 
 ## 0.5.0
 
-* add BLS support
-* add ability to create wallet from mnemonic or unsafe seed
-* add integration tests
-* LedgerContract switches path to optional
-* add address prefix to wallet
-* Add fallback hashlib for Ubuntu 22.04 LTS
+- add BLS support
```
Comment (5A11) [inline HISTORY.md:11]: The break lines added in this file seem random

**Comment by 5A11** [comment]
This tool seem to make several mistakes including not recognising the admonitions by mkdocs and some strange line breaking in History.md. Due to its automatic nature, markdown file lining needs to be 100% correct without needing manual changes which does not seem to be the case for this too. 

Therefore, unless through configurations we can get it working without any errors, I suggest we close this PR and add a tool in the future if the output is automatically 100% correct.

**Pull request closed without merging**